db/migrate/20160610124506_add_lives_on_to_middleware_server.rb

```diff
@@ -0,0 +1,11 @@
+class AddLivesOnToMiddlewareServer < ActiveRecord::Migration[5.0]
```

Comment:
On my first contacts with Rails, it wasn't that smart yet, I believe :) Thanks for the info!